frontend/src/routes/Upload.tsx

```diff
@@ -265,6 +292,13 @@ const UploadErrorBox: React.FC<{ error: unknown }> = ({ error }) => {
             probablyOurFault: true,
             potentiallyInternetProblem: false,
         };
+    } else if (error instanceof DOMException) {
```

Comment:
This is probably too broad an error class. Can we throw a custom error in the case of a cancellation as we do for all the other cases here? :thinking: 

See also my overall review comment, though.